lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1,5 +1,35 @@
 <?php
 
+/**
+ * @api {get} /wp-json/wp/v2/posts/:id Request Post Objects
+ * @apiName posts
+ * @apiGroup posts
+ *
+ * @apiParam {Number} id ID of the Post.
+ * @apiParam {String} [search] Keyword for search posts.
```

Comment:
These params are already defined elsewhere. I don't think we'll be adding them to the PHPDoc too.